src/defrag.c

```diff
@@ -61,17 +57,17 @@ int je_get_defrag_hint(void *ptr);
 void *activeDefragAlloc(void *ptr) {
     size_t size;
     void *newptr;
-    if (!je_get_defrag_hint(ptr)) {
+    if (!allocatorShouldDefrag(ptr)) {
```

Comment:
How does this file get access to allocatorShouldDefrag? I can't see that this file includes `allocator_defrag.h`. Neither does `server.h`. What am I missing?

Reply:
weird, was thinking it's in server.h... I wonder how it passed the tests..